.versionrc

```diff
@@ -0,0 +1,8 @@
+{
+    "bumpFiles": [
+        {
+            "filename": "VERSION.txt",
```

Comment:
Just a thought, I don't think we need the `.txt` extension. Just `VERSION` is fine. Feels cleaner for me that way for some reason 🤓.

Reply:
got it